CONTRIBUTING.md

```diff
@@ -92,22 +92,15 @@ A docker-compose.yml file is included in the repository that quickly and easily
       - Note that hot reloading is enabled for changes to front and backend source files
 
 #### Local HTTPS environment
-To develop and test with client certificate (CAC) authentication locally, SSL keys need to be generated locally.
-  1. navigate to /etc/hosts and add the following lines:
-      - `127.0.0.1	stage.solo.localhost`
-      - `127.0.0.1	api.stage.solo.localhost`
-      - `127.0.0.1	auth.stage.solo.localhost`
-  2. Create and change to a directory on your local system that will contain your keys. Here, the environment assumes they will be at the path referenced below, but can be changed via the `SSL_KEYS_VOLUME_ROOT` configuration setting.
-      - `mkdir -p ~/.ssh/solo`
-      - `cd ~/.ssh/solo`
-  3. Generate self-signed public and private keys
-      - `openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./nginx-selfsigned.key -out ./nginx-selfsigned.crt`
-      - `openssl dhparam -out ./dhparam.pem 1024`
-  4. Tell Chrome to allow self-signed certificates on localhost
-      - Navigate to `chrome://flags/#allow-insecure-localhost` and enable that setting
-  5. Execute docker-compose
+  1. After setting up the http environment, Navigate to solo/deploy/compose, make configureLocalEnv.sh executable, and run the script, set the Country name to US when prompted, skip the rest.
+      - `chmod +x configureLocalEnv.sh`
+      - `sudo sh configureLocalEnv.sh`
+      - `US` -> `ENTER` x6
```

Comment:
Is the chmod +x necessary if the file is committed as executable? Is there a way to script the config prompts for cert generation (maybe like [this](https://unix.stackexchange.com/a/104305))?

Reply:
These are great edits, see the recent commit.

Reply:
nicely done